GameOfThrones/Views/House/Index.cshtml

```diff
@@ -0,0 +1,30 @@
+﻿@model IEnumerable<GameOfThrones.Models.House>
+
+@{
+    Layout = "~/Views/Shared/_Layout.cshtml";
+}
+
+@{
+    ViewBag.Title = "Houses";
+}
+
+<h2>Houses</h2>
+<table class="table">
+    <tr>
+        <th>Name</th>
+        <th>Region</th>
+        <th>Words</th>
+        <th>Castle</th>
+        <th>Members</th>
```

Comment:
Could this use @Html.DisplayNameFor() instead?

Reply:
Html helpers are now used to render the table headers